src/components/dashboard/dashboard.xml

```diff
@@ -18,7 +18,7 @@
         <div
           t-foreach="getClickableCells()"
           t-as="clickableCell"
-          t-key="clickableCell.cell.id"
+          t-key="clickableCell.position.col + '-' + clickableCell.position.row"
```

Comment:
this will break on a symetric sheet structure

Reply:
I think that this will break in collaborative context if someone add a row/col before the cell.
Let's say there are two clickable cells:
- A2: key = `0-1`
- A3: key = `0-2`
Then, insert a row before 1
- A3 (was A2): key = `0-2` <-- same as old A2
- A4 (was A3): key = `0-3`